src/awx_plugins/credentials/aws_assumerole.py

```diff
@@ -0,0 +1,143 @@
+import datetime
+import hashlib
+
+from django.utils.translation import gettext_lazy as _
```

Comment:
FTR, a parallel effort moves such things into the interfaces repo temporarily. Although, the thing I was migrating was `gettext_noop`. I wonder if only one of the function variants needs to be used (noop or lazy)...

Anyway, at some point this will probably have to become
```suggestion
from awx_plugins.interfaces._temporary_private_api import gettext_noop
```
or
```suggestion
from awx_plugins.interfaces._temporary_private_api import gettext_lazy
```

Reply:
currently I think I have from `.plugin import CredentialPlugin, translate_function as _`. I will rewrite to point to the temp interface.

Reply:
The temporary private module has been renamed. Wait for a few hours to see the finalized version revealed.